core/src/main/java/hudson/tasks/BuildTrigger.java

```diff
@@ -367,6 +367,8 @@ public boolean showEvenIfUnstableOption(@CheckForNull Class<? extends AbstractPr
          * Form validation method.
          */
         public FormValidation doCheck(@AncestorInPath AbstractProject project, @QueryParameter String value) {
+            // JENKINS-32525: Check that it behaves gracefully for an unknown context
+            if (project == null) return FormValidation.ok();
```

Comment:
Would something like "Ancestor/Context Unknown: the project specified cannot be validated" be helpful?